task_1/UniqueItem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace task_1
+{
+    class UniqueItem
+    {
+        static int Id;
```

Comment:
better to have the following:

private static int _id:
public property - public int Id {get; private set}